package.json

```diff
@@ -11,7 +11,9 @@
   },
   "scripts": {
     "prepublish": "yarn run build",
-    "build": "rollup -c && yarn run minify-umd",
+    "build": "yarn run build:optimized && yarn run build:unoptimized",
```

Comment:
this can be `concurrently`, since one doesn't depend on the other

Reply:
I tried to use `concurrently 'yarn run build:optimized' 'yarn run build:unoptimized'`, but it threw up in the minify step. Maybe it was something with the `cat`? Or did I use it incorrectly?

Reply:
Accidental resolve. I think this happens because cat calls fflush on the license file in parallel which won’t work. You may want to prepend the license in a separate step using cat and piping to tee with the prepend option, then concurrently minify

Reply:
I'm going to skip this for now, but I'll keep in mind that it can be refactored better!